alfresco-search/src/main/java/org/alfresco/solr/AlfrescoCoreAdminHandler.java

```diff
@@ -838,6 +843,153 @@ private void actionREPORT(SolrQueryResponse rsp, SolrParams params, String cname
         }
     }
 
+    private void rangeCheck(SolrQueryResponse rsp,String cname) throws IOException
+    {
+        InformationServer informationServer = informationServers.get(cname);
+
+        Collection<Tracker> trackers = trackerRegistry.getTrackersForCore(cname);
+        MetadataTracker metadataTracker = null;
+        for(Tracker tracker : trackers)
+        {
+            if(tracker instanceof MetadataTracker)
+            {
+                metadataTracker = (MetadataTracker)tracker;
+            }
+        }
+
+        DocRouter docRouter = metadataTracker.getDocRouter();
+
+        if(docRouter instanceof DBIDRangeRouter) {
+
+            DBIDRangeRouter dbidRangeRouter = (DBIDRangeRouter) docRouter;
+            long startRange = dbidRangeRouter.getStartRange();
+            long endRange = dbidRangeRouter.getEndRange();
+
+            long maxNodeId = informationServer.maxNodeId();
+            long nodeCount = informationServer.nodeCount();
+
+            long bestGuess = -1;  // -1 means expansion cannot be done. Either because expansion
+                                 // has already happened or we're above safe range
+
+            long range = endRange - startRange; // We want this many nodes on the server
+            //System.out.println("##### Range:" + range);

```

Comment:
Remove System.out or add as debug loggin

Reply:
I'll remove all the commented System.out's in the final commit.